prod_ihor_usyk/domu.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+    <remote name="github" fetch="git://github.com" />
+    <remote name="yoctoproject" fetch="git://git.yoctoproject.org" />
+    <remote name="openembedded" fetch="git://git.openembedded.org" />
+    <remote name="linaro" fetch="git://git.linaro.org" />
+
+    <default remote="github" sync-j="10" sync-c="true" />
+
+    <project remote="epam" name="epmd-aepr/img-proprietary" path="proprietary" upstream="master" revision="master" />
+    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="rocko" revision="8af0b7d6e445b532088a068dc012757001be3a1f" />
+    <project remote="yoctoproject" name="poky" path="poky" upstream="rocko" revision="7e7ee662f5dea4d090293045f7498093322802cc" />
+    <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="rocko" revision="352531015014d1957d6444d114f4451e241c4d23" />
+    <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="rocko" revision="75dfb67bbb14a70cd47afda9726e2e1c76731885" />
+    <project remote="yoctoproject" name="meta-selinux" path="meta-selinux" upstream="morty" revision="7351dfc00d7a56b8982582ba8e10ffa8940932cb" />
+    <project name="kraj/meta-clang" path="meta-clang" upstream="rocko" revision="533f882e77714a231a1cef985fd9bb7f56e44145" />
```

Comment:
Revision is incorrect see prod-devel manifest file.

Reply:
Whole manifest file is incorrect.

Reply:
there is just a difference - this domu has img-proprietary
that is it 
below the body form prod-devel
<remote name="github" fetch="git://github.com" />
    <remote name="yoctoproject" fetch="git://git.yoctoproject.org" />
    <remote name="openembedded" fetch="git://git.openembedded.org" />
    <remote name="linaro" fetch="git://git.linaro.org" />

    <default remote="github" sync-j="10" sync-c="true" />

    <project name="renesas-rcar/meta-renesas" path="meta-renesas" upstream="rocko" revision="8af0b7d6e445b532088a068dc012757001be3a1f" />
    <project remote="yoctoproject" name="poky" path="poky" upstream="rocko" revision="7e7ee662f5dea4d090293045f7498093322802cc" />
    <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="rocko" revision="352531015014d1957d6444d114f4451e241c4d23" />
    <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="rocko" revision="75dfb67bbb14a70cd47afda9726e2e1c76731885" />
    <project remote="yoctoproject" name="meta-selinux" path="meta-selinux" upstream="morty" revision="7351dfc00d7a56b8982582ba8e10ffa8940932cb" />
    <project name="kraj/meta-clang" path="meta-clang" upstream="rocko" revision="533f882e77714a231a1cef985fd9bb7f56e44145" />